### ISSUE TITLE
Pip install fails on 0.14

### ISSUE THREAD

**Issue by adriaanbaelus** [okigan/awscurl#49]
Our CI failed today due to an update of awscurl.
Looking at https://pypi.org/project/awscurl/0.14/#history I see you've released 0.13 and 0.14 (not committed to repo yet, I think?), but when I try to install awscurl now (either locally or on CI server), I get this:

```
⚡ pip install awscurl
Collecting awscurl
  Downloading https://files.pythonhosted.org/packages/0f/c3/82ffdac5671b1f89f369c0c33aebd4122df3f4ab3c11c802bab41c2edb37/awscurl-0.14.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/zl/dqtyzbh92jl4v53l2gpxfzyh0000gp/T/pip-build-BjlhKY/awscurl/setup.py", line 5, in <module>
        with open("README.md", "r") as fh:
    IOError: [Errno 2] No such file or directory: 'README.md'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/zl/dqtyzbh92jl4v53l2gpxfzyh0000gp/T/pip-build-BjlhKY/awscurl/
```

**Comment by okigan** [comment]
@adriaanbaelus resolved in v0.15, please confirm resolution. 

Sorry for the issue 😯. 
 
please lock the version of awscurl that is used in your builds and upgrade it in a controlled manner (with requirements.txt, pip install awscurl==0.15 or similar)

**Comment by okigan** [comment]
closing due to inactivity.
